test/DotNetWorkerTests/GrpcFunctionDefinitionTests.cs

```diff
@@ -173,5 +214,11 @@ private class MyBlobFunctionClass
             }
         }
 
+        private class MyQueueFunctionClass
```

Comment:
Adding here as a reminder. We will need test in `FunctionMetadataGenerator.cs` for queue similar to Blob as a number of conditions decide if `SupportsDeferredBinding` is `true`

https://github.com/Azure/azure-functions-dotnet-worker/blob/cb93c716190b75dea3d85edb4344b06a156ee267/test/FunctionMetadataGeneratorTests/FunctionMetadataGeneratorTests.cs#L247